_demos/mouse.go

```diff
@@ -135,6 +135,7 @@ func main() {
 	btnfmt := "Buttons: %s"
 	keyfmt := "Keys: %s"
 	pastefmt := "Paste: [%d] %s"
+	focusfmt := "Focus: %s"
```

Comment:
I added this to `mouse.go` just because it was convenient. I could create a separate `focus.go` demo if necessary.

Reply:
No, I think this is fine.